docs/xml/elements/hours_open.rst

```diff
@@ -1,57 +1,103 @@
 
-   Document HoursOpen & Schedule
-
-+----------+--------------------+------------+----------+----------------------+----------------------------+
-| Tag      | Data Type          | Required?  | Repeats? |Description           |Error Handling              |
-|          |                    |            |          |                      |                            |
-+==========+====================+============+==========+======================+============================+
-| Schedule | Schedule           | Optional   | Repeats  |                      |                            |
-+----------+--------------------+------------+----------+----------------------+----------------------------+
++----------+--------------------+------------+----------+-----------------------+-----------------------------------+
+| Tag      | Data Type          | Required?  | Repeats? |Description            |Error Handling                     |
+|          |                    |            |          |                       |                                   |
++==========+====================+============+==========+=======================+===================================+
+| Schedule | Schedule           |**Required**| Repeats  |Defines a block of days| At least one valid `Schedule`_    |
+|          |                    |            |          |and hours that a place | must be present for ``HoursOpen`` |
+|          |                    |            |          |will be open.          | to be valid. If no valid          |
+|          |                    |            |          |                       | `Schedule`_ is present, the       |
+|          |                    |            |          |                       | implementation is required to     |
+|          |                    |            |          |                       | ignore the ``HoursOpen`` element. |
++----------+--------------------+------------+----------+-----------------------+-----------------------------------+
 
 HoursOpen.Schedule
 ------------------
 
+A sub-portion of the schedule. This describe a range of days, along with one or
+more set of open and close times for those days, as well as the options
+describing whether or not appointments are necessary or possible.
+
 +---------------------+------------------+-----------+----------+----------------------+----------------------------+
 | Tag                 | Data Type        | Required? | Repeats? |Description           |Error Handling              |
 |                     |                  |           |          |                      |                            |
 +=====================+==================+===========+==========+======================+============================+
-| Hours               | Hours            | Optional  | Repeats  |                      |                            |
+| Hours               | Hours            | Optional  | Repeats  |Blocks of hours in the|If the field is invalid or  |
```

Comment:
Could be helpful as an in-page anchor link by using \`Hours\`_ and changing the "Hours" header (same reasons as above).